docs/doc_sources/api_reference/index.rst

```diff
@@ -0,0 +1,34 @@
+.. _api_reference:
+
+=============
+API Reference
+=============
+
+The package ``dpctl`` provides
+
+* Python API
+   - :ref:`API objects <dpctl_pyapi>` in :py:mod:`dpctl` namespace
+   - :ref:`API objects <dpctl_tensor_pyapi>` in :py:mod:`dpctl.tensor` namespace
+   - :ref:`API objects <dpctl_memory_pyapi>` in :py:mod:`dpctl.memory` namespace
+   - :ref:`API objects <dpctl_program_pyapi>` in :py:mod:`dpctl.program` namespace
+   - :ref:`API objects <dpctl_utils_pyapi>` in :py:mod:`dpctl.utils` namespace
+* C API
```

Comment:
These two should be individual bullet points.

* `libsyclinterface` : A C wrapper for a subset of DPC++'s SYCL runtime.
* `dpctl_capi`: A C-API for the Python classes in the Python `dpctl` module